test/unit/metrics/test_metrics_engine_dicts.py

```diff
@@ -88,6 +88,27 @@
     metrics.log_loss_group_min: 7.67546133,
 }
 
+classification_keys = [
+    metrics.true_positive_rate_group_summary,
+    metrics.true_negative_rate_group_summary
+]
+
+metric_group_summary_pos_label_keys = [
+    metrics.true_positive_rate_group_summary,
+    metrics.true_negative_rate_group_summary,
+    metrics.false_positive_rate_group_summary,
+    metrics.false_negative_rate_group_summary,
+    metrics.selection_rate_group_summary,
+    metrics.precision_score_group_summary,
+    metrics.recall_score_group_summary,
+    metrics.f1_score_group_summary
+]
+
+# Subset of metrics which accept a pos_label argument
```

Comment:
I don't think we need this dictionary.

Reply:
Why not? I use it as a quick way of getting a subset of results which can be tested with fiddled `pos_label`

Reply:
Isn't that goal achieved by `metric_group_summary_pos_label_keys`?

Reply:
Good point, given that the 'data-driven' part is over the keys of the dictionary.